cram_pr2/cram_pr2_projection/src/low-level.lisp

```diff
@@ -266,8 +266,13 @@
 
 (defparameter *gripper-length* 0.2 "PR2's gripper length in meters, for calculating TCP -> EE")
 
+(defparameter *l-tcp-p* nil)
+(defparameter *r-tcp-p* nil)
+(defparameter *ee-poses* nil)
```

Comment:
So this is not used anymore, was is just for debugging?

Reply:
Debugging stuff is removed.